spec/controllers/events_spec.rb

```diff
@@ -22,7 +22,7 @@
     end
 
     describe 'name_and_id' do
-      include_context :open_to_np_associate, :get, :name_and_id, nonprofit_id: :__our_np
+      include_context :open_to_np_associate, :get, :name_and_id, nonprofit_id: :__our_np, with_status: 200
```

Comment:
Normally, we expect there to a be a 204 on success for authorization. Sometimes that doesn't happen. This tells the test "200 is okay instead!"

Reply:
nit - but I'd prefer the symbol `:ok` instead of 200....

Reply:
I'd generally agree but here we are.

Reply:
That's a good idea. I don't think now is the best time to address it though.